package.json

```diff
@@ -25,15 +25,15 @@
     "react": "^16.0.0"
   },
   "devDependencies": {
-    "@types/enzyme": "3.1.13",
+    "@types/assert": "^1.4.1",
```

Comment:
do we need this dependency?

Reply:
Yes, removal of @types/node removed the ambient assert definition

Reply:
Is some package depends on @types/assert?